### PR TITLE
Added PR template

### PR DESCRIPTION
**Pull request by percygautam**
Added temporary PR template till multiple PR template feature is added to GitHub.

### CODE REVIEW

**Comment by OriolAbril** [comment]
I think it has to be outside the `PULL_REQUEST_TEMPLATE` directory for GitHub to use it automatically.

**Comment by percygautam** [comment]
Yes, I also verified it [here](https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates). Thanks!

**Comment by OriolAbril** [comment]
I would use the new feature template, there is no need for a forth one. We should be able to get the type of change from the description, and its checklist is more complete (contributors can erase the ones that do not apply)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,17 +18,14 @@ consider the following points:
...
-- [ ] New feature
 
 ## Checklist
 <!-- Feel free to remove check-list items aren't relevant to your change -->
 
 - [ ] Does the PR follow [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
       PR format?
+- [ ] Has included a sample plot to visually illustrate the changes? (only for plot-related functions)
+- [ ] Is the new feature properly documented with an example?
 - [ ] Does the change introduced by this pull request are covered by existing or newly introduced tests.(using [pytest fixture pattern](
```
Comment (OriolAbril) [inline .github/PULL_REQUEST_TEMPLATE.md]: I prefer the wording in new-feature.md: `Does the PR include new or updated tests to cover the new feature (using pytest fixture pattern)?`

**Comment by percygautam** [comment]
@OriolAbril I have used the feature template and changed some things to generalize the template.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -18,17 +18,14 @@ consider the following points:
...
       https://docs.pytest.org/en/latest/fixture.html#fixture))?
 - [ ] Is your code style correct (follows pylint and black guidelines)?
```
Comment (OriolAbril) [inline .github/PULL_REQUEST_TEMPLATE.md]: maybe we can remove the `your` to make it less personal?

Comment (percygautam) [reply .github/PULL_REQUEST_TEMPLATE.md]: I have also removed it from other templates.

**Pull request merged**